frontend/talentsearch/src/js/components/search/EstimatedCandidates.tsx

```diff
@@ -6,59 +6,74 @@ interface EstimatedCandidatesProps {
+    const intl = useIntl();
 
-  function weight(msg: string) {
-    return updatePending ? (
-      <span className="lds-dual-ring" />
-    ) : (
-      <span data-h2-font-weight="b(800)">{msg}</span>
-    );
-  }
+    function weight(msg: string) {
+      return <span data-h2-font-weight="b(800)">{msg}</span>;
+    }
 
-  return (
-    <div
-      data-h2-bg-color="b(white)"
-      data-h2-border="b(lightgray, all, solid, s)"
-      data-h2-shadow="b(m)"
-    >
+    return (
       <div
-        data-h2-bg-color="b(linear-70[lightpurple][lightnavy])"
-        data-h2-padding="b(top-bottom, s) b(left, s) b(right, s)"
+        data-h2-bg-color="b(white)"
+        data-h2-border="b(lightgray, all, solid, s)"
+        data-h2-shadow="b(m)"
       >
-        <p
-          data-h2-font-color="b(white)"
-          data-h2-font-size="b(h4)"
-          data-h2-margin="b(all, none)"
-          data-h2-font-weight="b(700)"
+        <div
+          data-h2-bg-color="b(linear-70[lightpurple][lightnavy])"
+          data-h2-padding="b(top-bottom, s) b(left, s) b(right, s)"
         >
-          {intl.formatMessage({
-            defaultMessage: "Estimated Candidates",
-            description:
-              "Heading for total estimated candidates box next to search form.",
-          })}
-        </p>
-      </div>
-      <div data-h2-margin="b(top-bottom, m) b(right-left, l)">
-        <p data-h2-text-align="b(center)">
-          {intl.formatMessage(
-            {
-              defaultMessage: `{candidateCount, plural, =1 {There is approximately <weight>{candidateCount}</weight> candidate right now who meets your criteria.} other {There are approximately <weight>{candidateCount}</weight> candidates right now who meet your criteria.}}`,
+          <p
+            data-h2-font-color="b(white)"
+            data-h2-font-size="b(h4)"
+            data-h2-margin="b(all, none)"
+            data-h2-font-weight="b(700)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Estimated Candidates",
               description:
-                "Message for total estimated candidates box next to search form.",
-            },
-            {
-              weight,
-              candidateCount,
-            },
-          )}
-        </p>
+                "Heading for total estimated candidates box next to search form.",
+            })}
+          </p>
+        </div>
+        <div
+          data-h2-margin="b(top-bottom, m) b(right-left, l)"
+          aria-live="polite"
+        >
+          <p data-h2-text-align="b(center)">
+            {updatePending ? (
+              <span className="lds-dual-ring">
+                <span data-h2-visibility="b(invisible)">
+                  {intl.formatMessage({
+                    defaultMessage: "Searching...",
+                    description:
+                      "Message to display when a search is in progress.",
+                  })}
+                </span>
+              </span>
```

Comment:
Thoughts on pulling these out into `spinner` function to make the logic more legible? (I'm trying to figure out my own balance of using ["Thinking in React"](https://reactjs.org/docs/thinking-in-react.html) style of componentization, but without so much overhead of putting them all in files, so I personally try to lean on little pseudo-component functions to bundle up markup. It's ok if you feel differently!)

```js
function spinner() (
  <span className="lds-dual-ring">
    <span data-h2-visibility="b(invisible)">
      {intl.formatMessage({
        defaultMessage: "Searching...",
        description:
          "Message to display when a search is in progress.",
      })}
    </span>
  </span>
)

Reply:
Great suggestion, the spinner has been moved to its own component for readability and reusability. 

[move spinner, add story, change candidate weight](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2620/commits/e5e0e3bc4eaf184123557ed90bfb0299fc3278a4)